### ISSUE TITLE
EF6.4 Add-Migration fails for multi-targeted project

### ISSUE THREAD

**Issue by phekmat** [dotnet/ef6#1568]
`Add-Migration` (and presumably its friends) fail from 6.3 on in the PMC if the host project for the migrations targets both netstandard and full framework.

`EntityFramework6.psm1` only takes into consideration the `TargetFrameworkMoniker` rather than `TargetFrameworkMonikers` (if available)

```
Exception message: The Entity Framework Package Manager Console Tools don't support this framework
```

### Steps to reproduce
Create a project with `<TargetFrameworks>net461;netstandard2.1</TargetFrameworks>` and attempt to `Add-Migration`

### Further technical details
EF version: 6.4.0
Operating system: Windows 10
IDE: Visual Studio 2019 16.4.3

**Comment by PhilPJL** [comment]
@phekmat 

Hi, I've had problems like this before, but it's currently all working fine for me

EF version: 6.4.0
Operating system: Windows 10
IDE: Visual Studio 2019 16.4.5

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFrameworks>net461;netstandard2.1</TargetFrameworks>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="EntityFramework" Version="6.4.0" />
  </ItemGroup>
  
  <PropertyGroup>
    <EmbeddedResourceUseDependentUponConvention>true</EmbeddedResourceUseDependentUponConvention>
  </PropertyGroup>

</Project>
```

**Comment by phekmat** [comment]
@PhilPJL thanks for that. It's working again for me as well :man_shrugging:. VS may have been caching the active configuration (or something else was fixed in 16.4.4 or 16.4.5) as `net461` has to come first when multi-targeting with `netstandard`. 

The Powershell module could perhaps take that into account so the order doesn't matter, but I'll close this since that's a different issue.
